src/subramanium/Assignment23.java

```diff
@@ -0,0 +1,40 @@
+//Write a program to find count of Uppercase, lowercase, digits and special characters.
+
+package subramanium;
+
+public class Assignment23 {
+	void findCharCase(String input) {
+		int ucount=0;
+		int lcount=0;
+		int dcount=0;
+		int special=0;
+		for(int index=0; index<input.length();index++) {
+			if(Character.isUpperCase(input.charAt(index))){
```

Comment:
you can write this as char ch = input.charAt(index); outside all condition statement, so you no need to rewrite it again and again, you just need to write like Character.isUpperCase(ch)